templates/dydns_egi_update.yml

```diff
@@ -50,8 +51,6 @@ topology_template:
                 get_input: nsupdate_secret
               new_ip:
                 get_attribute: [ simple_node, public_address, 0 ]
-      requirements:
-        - host: simple_node
```

Comment:
These lines must not be removed otherwise this node will not be applied to any compute node.
To apply to the htcondor cluster template you must create another dydns template setting as host "lrms_server".

Reply:
so a "super class" cannot be used that includes both `simple_node` and `lrm_server`, right?

Reply:
ok I added now the requested changes.